app/src/main/java/com/github/se/gomeet/ui/mainscreens/Explore.kt

```diff
@@ -324,62 +327,65 @@ private fun ContentInColumn(
-                                          crossfade(true)
-                                          placeholder(R.drawable.gomeet_logo)
-                                        })
-                                .build())
-                      } else {
-                        painterResource(id = R.drawable.gomeet_logo)
-                      }
-                  Image(
-                      painter = painter,
-                      contentDescription = "",
-                      modifier = Modifier.fillMaxSize(),
-                      alignment = Alignment.Center,
-                      contentScale = ContentScale.Crop)
+      LazyColumn(
+          modifier = Modifier.alpha(columnAlpha),
+          state = listState,
+          horizontalAlignment = Alignment.CenterHorizontally) {
+            itemsIndexed(events) { _, event ->
+              Column {
+                Card(
+                    elevation = 4.dp,
+                    modifier =
+                        Modifier.size(width = 360.dp, height = 200.dp).padding(8.dp).clickable {
+                          nav.navigateToEventInfo(
+                              eventId = event.eventID,
+                              title = event.title,
+                              date = getEventDateString(event.date),
+                              time = getEventTimeString(event.time),
+                              description = event.description,
+                              organizer = event.creator,
+                              loc = LatLng(event.location.latitude, event.location.longitude),
+                              rating = 0.0 // TODO: replace with actual rating
+                              // TODO: add image
+                              )
+                        }) {
+                      val painter: Painter =
+                          if (event.images.isNotEmpty()) {
+                            rememberAsyncImagePainter(
+                                ImageRequest.Builder(LocalContext.current)
+                                    .data(data = event.images[0])
+                                    .apply(
+                                        block =
+                                            fun ImageRequest.Builder.() {
+                                              crossfade(true)
+                                              placeholder(R.drawable.gomeet_logo)
+                                            })
+                                    .build())
+                          } else {
+                            painterResource(id = R.drawable.gomeet_logo)
+                          }
+                      Image(
+                          painter = painter,
+                          contentDescription = "",
+                          modifier = Modifier.fillMaxSize(),
+                          alignment = Alignment.Center,
+                          contentScale = ContentScale.Crop)
```

Comment:
personal opinion: make the image's corner rounded and remove the shadow to have a cleaner and more modern look